doc/ref/string.xml

```diff
@@ -147,10 +147,10 @@ next line
 There are a number of <E>special character sequences</E> that can be used
 between the singlequotes of a character literal or between the
 doublequotes of a string literal to specify characters. 
-They consist of two characters.
-The first is a backslash <C>\</C>. The second may be any character.
-If it is an octal digit (from <C>0</C> to <C>7</C>) there must be two more
-such digits. The meaning is given in the following list
+They consist of a backslash <C>\</C> followed by a second character
+indicating the type of special character sequence, and possibly more characters.
+The following special character sequences are currently defined. Any other sequence
+starting with a backslash results in an error.
```

Comment:
If we can't agree on changing the documented and actual behaviour, then this last sentence is controversial. In that case, we should drop it, but can keep the rest of this change.